### ISSUE TITLE
Subtitles / lag time

### ISSUE THREAD

**Issue by xakiv** [popcorntime/popcorntime#133]
Operating System Version:
Windows 7 64bit
Popcorn Time Version:
0.3.9-0
Download date:
2-apr-16
#### Expected Behaviour

Default subtitles (not those added by user) are usually synchronised
#### Actual Behaviour

In some movies a found out a gap between sub's and the movie (e.i: "Withnail and I" / english sub)
I don't remember all the others movie title with the same issue.
#### Steps to repoduce the behaviour

1. Run "Withnail and I" in movie section
2. Choose english as sub's language
3. there will be a gap between sub's and movie

Let me know if I can help,
And thx for your commitment!

**Comment by Zerphix** [comment]
Hey xakiv, I tried to reproduce the error but I couldn't. However, I have experienced this behavior on the past with another movies/series. The root cause for this seems to be PCT has no control over the subtitles, it just fetches them from different sites. As a possible workaround, consider to use the following key combinations to sync the subtitles:

![captura de pantalla 2016-04-15 a las 10 27 44 a m](https://cloud.githubusercontent.com/assets/9393928/14566362/e9a3e97c-02f4-11e6-80df-6ed353425489.png)

If you want to review the whole key combinations available, press "shift + ?".
